### PR TITLE
Add placeholders in product settings page

### LINKED ISSUE #25105: Add placeholders in product settings page
#### Is your feature request related to a problem?

To help the users better fill the stock labels and the delivery times fields, we should add a placeholder for each field of the product settings page

#### Describe the solution you'd like

Add the following placeholders:

- Label of in-stock products: "In stock" in Admin.Shopparameters.Help
- Label of out-of-stock products with allowed backorders: "On backorder" in Admin.Shopparameters.Help
- Label of out-of-stock products with denied backorders: "Out of stock"  in Admin.Shopparameters.Help
- Delivery time of in-stock products: "Delivered within 3-4 days"  in Admin.Shopparameters.Help
- Delivery time of out-of-stock products with allowed backorders: "Delivered within 5-7 days"  in Admin.Shopparameters.Help

### PR DESCRIPTION
**Pull request by ghost**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add placeholders in product settings page
| Type?             |  improvement
| Category?         | BO
| BC breaks?        |  no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/25105
| How to test?      | Check if the page have placeholders 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26055)

### CODE REVIEW

**Review by NeOMakinG — DISMISSED** [review]
Hope wording are fine!

**Review by matks — DISMISSED** [review]
Approved for logic. Waiting for @Julievrz review for wording.

File: src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/ProductPreferences/StockType.php [hunk 1]
```diff
@@ -51,22 +51,47 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('in_stock_label', TranslatableType::class, [
                 'type' => TextType::class,
                 'only_enabled_locales' => false,
+                'options' => [
+                    'attr' => [
+                        'placeholder' => $this->trans('In Stock', 'Admin.Shopparameters.Help'),
```
Comment (Julievrz) [inline src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/ProductPreferences/StockType.php]:
```suggestion
                        'placeholder' => $this->trans('In stock', 'Admin.Shopparameters.Help'),
```

**Review by Julievrz — COMMENTED** [review]
Just a little thing. 

Thanks in advance. ☺️

**Comment by ghost** [comment]
Done @Julievrz

**Review by NeOMakinG — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by florine2623** [comment]
Hello @okom3pom ,

Thanks for the PR ! Looks good to me !

**Comment by Progi1984** [comment]
Thanks @okom3pom & @florine2623

**Pull request merged**